analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs

```diff
@@ -55,7 +55,9 @@ public abstract class GeneratedCodeRecognizer
         protected abstract string GetAttributeName(SyntaxNode node);
 
         public bool IsGenerated(SyntaxTree tree) =>
```

Comment:
I would consider renaming this method since it does not only check for generated files anymore.
Maybe `IsConsideredGenerated`?